lib/haml_lint/tree/tag_node.rb

```diff
@@ -9,7 +9,7 @@ class TagNode < Node
     #
     # @return [Array<String>]
     def dynamic_attributes_sources
-      @value[:attributes_hashes]
+      @value.fetch(:attributes_hashes, [])
```

Comment:
🤔  Do you know if support for this was dropped or if it was moved? I worry that we'll miss some things if we just default this to an empty array.